params.py

```diff
@@ -32,19 +32,21 @@
     }
 }
 
-
 """ 
 New datasets can be added to 'all'.
 For the input of an experiment the 'current_set_up' list can be updated.
+Datasets input options:
+- 'datasets' : `Dataset` # The dataset is split into calibration and test pairs
```

Comment:
this is really hard to get from the code. Maybe a wrapper like
Data(calibrate=(dataset1, dataset2), test=(dataset3))
this would allow extensions later like
Data(calibrate=(dataset1, dataset2), test=(dataset3), calibrate_test = ((dataset4, test_size=0.8),))

or just specify for each set the fraction to go to testing
((dataset1, .8))
((dataset1, 1), (dataset2, 1), (dataset3, 0))